client/my-sites/checkout/checkout-thank-you/domain-mapping-details.jsx

```diff
@@ -10,128 +10,52 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import PurchaseDetail from 'calypso/components/purchase-detail';
-import { isSubdomain } from 'calypso/lib/domains';
+import { getSelectedDomain, isSubdomain } from 'calypso/lib/domains';
 import { isBusiness } from 'calypso/lib/products-values';
-import { MAP_EXISTING_DOMAIN, MAP_SUBDOMAIN } from 'calypso/lib/url/support';
 import { getSelectedSite } from 'calypso/state/ui/selectors';
+import QuerySiteDomains from 'calypso/components/data/query-site-domains';
+import { getDomainsBySiteId, isRequestingSiteDomains } from 'calypso/state/sites/domains/selectors';
+import DomainMappingInstructions from 'calypso/my-sites/domains/components/mapping-instructions';
 
 const DomainMappingDetails = ( {
 	domain,
```

Comment:
Maybe to avoid confusion on whether `domain` is the domain object or string, you could rename it here like this:

```suggestion
	domain: domainName,
```